kubernetes-kit-starter/src/main/java/com/vaadin/kubernetes/starter/sessiontracker/SessionSerializer.java

```diff
@@ -272,7 +272,7 @@ private void queueSerialization(String sessionId,
         // sense to retry the operation instantly.
         CompletableFuture.runAsync(
                 () -> backendConnector.markSerializationStarted(clusterKey),
-                executorService).whenComplete((unused, error) -> {
+                executorService).handle((unused, error) -> {
                     if (error != null) {
                         pending.remove(sessionId);
```

Comment:
If the `sessionId` is removed in the `whenComplete()` stage, does it need to done here too?

Reply:
Good point. This line can be removed

Reply:
done